examples/simple/stdout_exporter.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <iostream>
+
+namespace trace    = opentelemetry::trace;
+namespace nostd    = opentelemetry::nostd;
+namespace sdktrace = opentelemetry::sdk::trace;
+
+class StdoutExporter final : public sdktrace::SpanExporter
+{
+  std::unique_ptr<sdktrace::Recordable> MakeRecordable() noexcept
+  {
+    return std::unique_ptr<sdktrace::Recordable>(new sdktrace::SpanData);
+  }
+
+  sdktrace::ExportResult Export(nostd::span<std::unique_ptr<sdktrace::Recordable>> &spans) noexcept
```

Comment:
How come none of this is const?

Reply:
I changed the span to `const` (although I think it wouldn't matter much, as the processor sees the content of the span as invalidated anyway after the call to `Export`).

Was this the intent of your question? Or do you also think the recordable itself should be `const`?